validator/gossip/signed_object.py

```diff
@@ -260,30 +260,30 @@ def sign_object(self, signingkey):
         self._identifier = hashlib.sha256(self.Signature).hexdigest()
 
     def serialize(self, signable=False):
-        """Generates a CBOR serialized dict containing the a SignatureKey
+        """Generates a CBOR serialized dict containing the a SignatureName
```

Comment:
Delete extra article "a" in this comment: Generates a CBOR serialized dict containing the a SignatureName